src/components/rangeInput.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from "react"
+
+import Input from "./input/index"
+
+import Label from "./fieldHelpers/label"
+
+interface Props {
+  name: {
+    min: string
+    max: string
+  }
+  handleChangeMin: (value) => void
+  handleChangeMax: (value) => void
```

Comment:
I noticed in related project PRs that you're passing the same change handler. Is the separation here really worth it then?

Reply:
No, we don't need both !